Package.swift

```diff
@@ -67,18 +67,15 @@ let package = Package(
                 .product(name: "FluentSQLiteDriver", package: "fluent-sqlite-driver"),
             ],
             resources: [
-                .copy("Templates"),
+                .copy("Templates")
             ],
             swiftSettings: swiftSettings
         ),
     ]
 )
 
-var swiftSettings: [SwiftSetting] { [
-    .enableUpcomingFeature("ExistentialAny"),
-    .enableUpcomingFeature("ConciseMagicFile"),
-    .enableUpcomingFeature("ForwardTrailingClosures"),
-    .enableUpcomingFeature("DisableOutwardActorInference"),
-    .enableUpcomingFeature("StrictConcurrency"),
-    .enableExperimentalFeature("StrictConcurrency=complete"),
-] }
+var swiftSettings: [SwiftSetting] {
+    [
+        .enableUpcomingFeature("ExistentialAny")
```

Comment:
You could also throw this in there as well:
```
        /// https://github.com/apple/swift-evolution/blob/main/proposals/0413-typed-throws.md
        /// Enable the full potential of typed throws.
            .enableUpcomingFeature("FullTypedThrows"),
```
Not enabled in Swift 6. If you use an already-enabled flag, Swift 6 will throw errors and won't let the CI pass.